hoot-core/src/main/cpp/hoot/core/conflate/extractors/AttributeDistanceExtractor.h

```diff
@@ -34,6 +34,7 @@ namespace hoot
 
 /**
  * See exporatory funds report for details.
+ * Compares "distance" between tags
```

Comment:
Thanks for updating.  Could you also throw this text in the description return method for the class?